lib/web_ui/lib/src/engine/text_editing/input_type.dart

```diff
@@ -14,7 +14,7 @@ part of engine;
 abstract class EngineInputType {
   const EngineInputType();
 
-  static EngineInputType fromName(String name) {
+  static EngineInputType fromName(String? name) {
```

Comment:
`name` shouldn't be null. It always get defaulted to `TextInputType.text` on the framework side.

Reply:
Done.